app/components/my-map/show-all-observations/component.js

```diff
@@ -34,17 +34,18 @@ export default class MyMapShowAllObservationsComponent extends Component {
 
   @action
   markerTooltipOpen(observation) {
+    const { id } = observation;
```

Comment:
Almost perfect 😄 How about to do that one row higher where you have `observation`?